lib/src/components/NxBinaryDonutChart/__tests__/NxDonutChart.test.tsx

```diff
@@ -5,20 +5,73 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import NxBinaryDonutChart from '../NxBinaryDonutChart';
+import NxBinaryDonutChart, {Props} from '../NxBinaryDonutChart';
 
 describe('NxBinaryDonutChart', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxBinaryDonutChart, {percent: 0});
+  const minimalProps: Props = {
+        value: 0
+      },
+      propsComponent = enzymeUtils.getShallowComponent<Props>(NxBinaryDonutChart, minimalProps);
+
+  const minimalDeprecatedProps: Props = {
+        percent: 0
+      },
+      DeprecatedPropsComponent = enzymeUtils.getShallowComponent<Props>(NxBinaryDonutChart, minimalDeprecatedProps);
+
+  it('sets the role to "meter"', function() {
+    expect(propsComponent()).toHaveProp('role', 'meter');
+    expect(DeprecatedPropsComponent()).toHaveProp('role', 'meter');
+  });
+
+  it('sets the aria-valuemin to 0', function() {
+    expect(propsComponent()).toHaveProp('aria-valuemin', 0);
+    expect(DeprecatedPropsComponent()).toHaveProp('aria-valuemin', 0);
+  });
+
+  it('sets the aria-valuemax to 100 when maxVal is not specified', function() {
+    expect(propsComponent()).toHaveProp('aria-valuemax', 100);
+    expect(DeprecatedPropsComponent()).toHaveProp('aria-valuemax', 100);
+  });
+
+  it('sets the aria-valuemax to maxVal when maxVal is specified', function() {
+    expect(propsComponent({maxVal: 200})).toHaveProp('aria-valuemax', 200);
+    expect(DeprecatedPropsComponent({maxVal: 200})).toHaveProp('aria-valuemax', 200);
+  });
+
+  it('sets the aria-valuenow to value', function() {
+    expect(propsComponent({value: 30})).toHaveProp('aria-valuenow', 30);
```

Comment:
Add another line that checks `aria-valuenow` with a non-default `maxVal`

Reply:
When the `value` is used, it is applied to aria-valuenow whether the maxVal is specified or not.
When the `percent` is used, the calculated value based on `percent ` and default maxVal/ non-default maxVal will be assigned to `aria-valuenow`. There are unit test for these under `percent` describe block. 
2b591aae733e6fe31f0544d52ab760c662ce85cf